test/serializers/serializer_for_test.rb

```diff
@@ -27,6 +27,16 @@ def test_overwritten_serializer_for_array
       end
 
       class SerializerTest < Minitest::Test
+        module ResourceNamespace
+          class Post < ::Model
```

Comment:
I strongly disagree in the context of a class this is neither a semaphore nor struct nor exception object.